src/sql/base/browser/ui/panel/panel.component.ts

```diff
@@ -52,8 +52,10 @@ let idPool = 0;
 					</div>
 				</div>
 			</div>
-			<div class="tab-content fullsize">
-				<ng-content class="fullsize"></ng-content>
+			<div class="tab-content">
+				<div class="fullsize" style="position: absolute">
```

Comment:
>class [](start = 9, length = 5)

Can you check if all panels that we have look fine? no regression?

Reply:
i checked query output, dashboard, and restore and they all seem to work

Reply:
Can you also the left nav bar works?